### DISCUSSION TITLE
Homebrew formulae

### DISCUSSION THREAD

**Discussion by AkihiroSuda** [WasmEdge/WasmEdge#3096] [Ideas]
FYI, here is my PR for Homebrew to allow `brew install wasmedge` 
- https://github.com/Homebrew/homebrew-core/pull/157617

Plugins such as https://wasmedge.org/docs/contribute/source/plugin/wasi_nn/ will be separate formulae, due to the huge amount of deps.

**Comment by alabulei1** [comment]
Thank you so much. @AkihiroSuda
